core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -212,7 +212,7 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
     executionContext: ExecutionContext
   ): Future[ExecutionServiceStatusResponse] = {
     val submissionFuture = datasource.inTransaction { dataAccess =>
-      dataAccess.submissionQuery.loadSubmission(submissionId)
+      dataAccess.uniqueResult[SubmissionRecord](dataAccess.submissionQuery.findById(submissionId))
```

Comment:
Note that `uniqueResult[SubmissionRecord](findById(submissionId))` is the first line of `loadSubmission`, so this is a good replacement.